mathosphere-core/src/main/java/com/formulasearchengine/mathosphere/mathpd/FlinkPd.java

```diff
@@ -38,6 +40,13 @@ public static void run(FlinkPdCommandConfig config) throws Exception {
                     @Override
                     public void reduce(Iterable<Tuple2<ArxivDocument, ArxivDocument>> iterable, Collector<Tuple2<Integer, String>> collector) throws Exception {
                         for (Tuple2<ArxivDocument, ArxivDocument> i : iterable) {
+                            NonWhitespaceNodeList allMathExpressionsF0 = i.f1.getMathTags();
+                            for(Node n : allMathExpressionsF0){
+                                Multiset<String> contentNodesF0 = i.f1.getCElements(n);
+                                System.out.println(contentNodesF0.toString());
```

Comment:
Please avoid using sysouts it fills up the server logs and slows donw processing.
For debugging on server we us log4j